examples/deck.gl/3d-tiles/tile-3d-layer.js

```diff
@@ -69,13 +69,17 @@ export default class Tile3DLayer extends CompositeLayer {
 
   async _loadTileset(tilesetUrl) {
```

Comment:
We should remove this class from examples and use deck.gl's Tile3DLayer

Reply:
Yes, I always keep them sync and will delete it before this PR is landed.